FLEDGE.md

```diff
@@ -664,7 +666,8 @@ The arguments to this function are:
 
 *   auctionSignals and perBuyerSignals: As in the call to `generateBid()` for the winning interest group.
 *   sellerSignals: The output of `reportResult()` above, giving the seller an opportunity to pass information to the buyer. In the case where the winning buyer won a component auction and then went on to win the top-level auction, this is the output of component auction's seller's `reportResult()` method.
-*   browserSignals: Similar to the argument to `reportResult()` above, though without the seller's desirability score, but with additional `seller`, `madeHighestScoringOtherBid` and potentially `interestGroupName` fields.
+*   browserSignals: Similar to the argument to `reportResult()` above, though without the seller's desirability score, but with additional `adCost`, `seller`, `madeHighestScoringOtherBid` and potentially `interestGroupName` fields.
+    *   The `adCost` field contains the value that was returned by `generateBid` (or the bid value if no value was specified), stochastically rounded to fit into a floating point number with an 8 bit mantissa and 8 bit exponent.
```

Comment:
Hm should this really have the bid value as the default?  What's the thinking here?  (I'd have guessed 0 or null or something?)

Reply:
My thought was that in the case of view-based ads (CPV), the ad cost would be the bid cost. But we could change the default to be any floating point number if that is preferable.

Reply:
Could we keep it as an optional field, and just omit it from `browserSignals` if it was not produced by `generateBid`?

Reply:
Sure. Changed to omit adCost if it was not returned by generateBid.